### PR TITLE
docs: basic troubleshooting page

### PR DESCRIPTION
**Pull request by mlegner**

### CODE REVIEW

File: docs/usage/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+```admonish tip title="Debug logging"
+You can enable debug logging for Walrus by setting the environment variable `RUST_LOG=walrus=debug`.
+```
+
+## Latest binary
+
+Before undertaking any other steps, make sure you have the [latest `walrus`
+binary](./setup.md#installation). If you have multiple versions in different locations, find the
+actually used binary with `which walrus`.
```
Comment (gdanezis) [inline docs/usage/troubleshooting.md:11]: I would say "the binary that will actually be used with" ...

File: docs/usage/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+The Walrus Devnet and Testnet are wiped periodically and require updating to the latest binary and
+configuration. If you get an error like "could not retrieve enough confirmations to certify the
+blob", you are probably using an outdated configuration pointing to an inactive Walrus system. In
+this case, update your configuration file with the latest [configuration](./setup.md#configuration)
+and make sure the CLI uses the intended configuration.
+
+```admonish tip
+The path to the Walrus configuration file used by the `walrus` client binary is printed at the
+beginning.
```
Comment (gdanezis) [inline docs/usage/troubleshooting.md:23]: At the beginning of what?

**Review by gdanezis — APPROVED** [review]
Nice, a couple of comments.

File: docs/usage/troubleshooting.md (hunk 2 above)
Comment (mlegner) [reply docs/usage/troubleshooting.md:23]: Will clarify.

**Pull request merged**